buildingmotif/api/views/template.py

```diff
@@ -69,18 +69,19 @@ def evaluate(template_id: int) -> flask.Response:
             "message": "request content type must be json"
         }, status.HTTP_400_BAD_REQUEST
 
-    model_id = request.get_json().get("modelId")
+    model_id = request.get_json().get("model_id")
     if model_id is None:
         return {"message": "body must contain 'model_id'"}, status.HTTP_400_BAD_REQUEST
     try:
-        model_id = Model.load(model_id)
+        model = Model.load(model_id)
     except NoResultFound:
         return {"message": f"No model with id {model_id}"}, status.HTTP_404_NOT_FOUND
 
     bindings = request.get_json().get("bindings")
     if bindings is None:
         return {"message": "body must contain 'bindings'"}, status.HTTP_400_BAD_REQUEST
     bindings = get_bindings(bindings)
+    bindings = {k: model.name + "/" + v for k, v in bindings.items()}
```

Comment:
Yeah, that's pretty much what I was saying. The issue is that if a model's URI ends with a separator we get two. If we did `model.name.strip("/") + "/" + v for ...` then we're guaranteed a single `/`